packages/flame/lib/src/game/mixins/game.dart

```diff
@@ -97,7 +96,13 @@ mixin Game {
   /// Check [AppLifecycleState] for details about the events received.
   void lifecycleStateChange(AppLifecycleState state) {}
 
-  /// Use for calculating the FPS.
+  /// Use for calculating the FPS in `HasFPS`.
+  void onPostFrameCallback(Duration duration) {}
+
+  /// Use for calculating the FPS in `FPSCounter`.
+  @Deprecated(
+    'This will be removed in favor of `onPostFrameCallback` in v1.2.0',
```

Comment:
```suggestion
    'This will be removed in favor of `onPostFrameCallback` in v1.3.0',
```
It will live with the deprecation tag in v1.2.0 and then be removed in v1.3.0